utils/convert.go

```diff
@@ -83,3 +86,55 @@ func ByteSize(bytes uint64) string {
 	result = strings.TrimSuffix(result, ".0")
 	return result + unit
 }
+
+// ToString Change arg to string
+func ToString(arg interface{}, timeFormat ...string) string {
+	if len(timeFormat) > 1 {
```

Comment:
can you remove these unnecessary lines, when someone passes multiple formats, they are not checked or taken into account